modules/custom/az_alphabetical_listing/az_alphabetical_listing.module

```diff
@@ -15,6 +20,64 @@ function az_alphabetical_listing_preprocess_views_view(&$variables) {
   }
 }
 
+/**
+ * Implements template_preprocess_views_view_summary()
+ */
+function az_alphabetical_listing_preprocess_views_view_summary(&$variables) {
+  // Add styling classes for az_alphabetical_glossary_attachment.
+  $view = $variables['view'];
+  $rows = $variables['rows'];
+  if ($view->id() === 'az_alphabetical_glossary' && $view->current_display === 'az_alphabetical_glossary_attachment') {
+    $views_field_alias_for_letter = 'node__field_az_alphabetical_index_field_az_alphabetical_inde';
+
+    // Create a quick lookup map from $view->result for easier access
+    // Keyed by the letter, value is the #existed_in_original flag.
+    $original_data_map = [];
+    foreach ($view->result as $result_row_from_post_execute) {
+      if (isset($result_row_from_post_execute->{$views_field_alias_for_letter})) {
+        $letter = $result_row_from_post_execute->{$views_field_alias_for_letter};
+        $existed = FALSE;
+        if (isset($result_row_from_post_execute->{'#existed_in_original'})) {
+          $existed = (bool) $result_row_from_post_execute->{'#existed_in_original'};
+        }
+        $original_data_map[$letter] = $existed;
+      }
+    }
```

Comment:
This loop seems like it can be avoided by usage of the `num_records` property present on the original rows - this essentially calculates the same information.